src/python/pants/backend/python/rules/BUILD

```diff
@@ -1 +1,5 @@
-python_library()
+python_library(
+  dependencies=[
+    'src/python/pants/backend/python/subsystems',
```

Comment:
Looks like this needs deps to satisfy `pants.engine.*` and `pants.rules.*` imports too.